recipe/meta.yaml

```diff
@@ -30,7 +30,7 @@ outputs:
         - python
         - alembic >=0.8.3,<0.9
         - bleach ==2.1.2
-        - configparser >=3.5.0, <3.6.0
+        - configparser >=3.5.0, <3.6.0  # [py<36]
```

Comment:
So think we don't need this on Python 3.5 either as it's a backport package from Python 3.5 to older Python version.

Reply:
I'll try it that way and try to reenable 3.5